### ISSUE TITLE
py.typed marker not included in the package

### ISSUE THREAD

**Issue by mdantonio** [fubuloubu/eth-keys#89]
* Version: 0.4.0
* Python: 3.10
* OS: osx

### What was wrong?

The package has type hints and has a py.typed marker, but it is not included when the package is installed. As a consequence mypy ignore the package with error:

```
Skipping analyzing "eth_keys": module is installed, but missing library stubs or py.typed marker
```

### How can it be fixed?

py.typed is included in package_data but I think that the package name is wrong:

This:
```
    package_data={'eth-keys': ['py.typed']},
```
should be:
```
    package_data={'eth_keys': ['py.typed']},
```

i.e. `eth-keys` -> `eth_keys`

**Comment by fselmo** [comment]
Thanks for reporting @mdantonio 👌. Seems simple enough. We should be able to get to this soon but if you'd like to submit a PR I think that might move things along quicker as well.

**Comment by mdantonio** [comment]
Sure!
It is done

**Comment by kclowes** [comment]
Closed with #90
